classes/Cart.php

```diff
@@ -431,7 +431,7 @@ public function getAverageProductsTaxRate(&$cartAmountTaxExcluded = null, &$cart
         if ($cart_vat_amount == 0 || $cartAmountTaxExcluded == 0) {
             return 0;
         } else {
-            return Tools::ps_round($cart_vat_amount / $cartAmountTaxExcluded, 3);
```

Comment:
This was 3 instead of 2 😛

Reply:
Mmm, probably because it's a tax rate, si it represents a percentage Maybe in this specific case the precision has to be different

For example if you have a 20.5% tax rate -> 0.205 you indeed need three digit precision The computing precision must be used for prices computation but not in this case

We can create a const to store this value though, like `Tax::DEFAULT_PRECISION` or something similar

Reply:
Feedback valid and applied in https://github.com/PrestaShop/PrestaShop/pull/20378/commits/da189d1ec824ed02d41f3aa4c475c6c187daccb4 👍